### PR TITLE
Internal Model Refactoring

### PR DESCRIPTION
**Pull request by Waguramu**
Add core service. Refactor view. Refactor map service. Add view service.

### CODE REVIEW

File: erdblick_app/app/debugapi.component.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import {uint8ArrayFromWasm} from "./wasm";
 import {Cartesian3} from "cesium";
+import {CoreService} from "./core.service";
+import {MapService} from "./map.service";
+import {FeatureLayerStyle} from "../../build/libs/core/erdblick-core";
+import {f} from "../../static/bundle/cesium/Workers/chunk-LYPPBP4Q";
```
Comment (josephbirkner) [inline erdblick_app/app/debugapi.component.ts:8]: tbr

File: erdblick_app/app/map.panel.component.ts [hunk 2]
```diff
@@ -215,18 +222,18 @@ export class MapPanelComponent {
 
     focus(tileId: bigint, event: any) {
         event.stopPropagation();
-        if (this.mapService.mapModel.getValue() && this.mapService.coreLib !== undefined) {
-            this.mapService.mapModel.getValue()!.zoomToWgs84PositionTopic.next(
-                this.mapService.coreLib.getTilePosition(BigInt(tileId))
+        if (this.coreService.coreLib !== undefined) {
```
Comment (josephbirkner) [inline erdblick_app/app/map.panel.component.ts:225]: These checks should not be necessary anymore. If they are, maybe add a shorthand lib() method to coreService.

File: erdblick_app/app/view.service.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import {Injectable} from "@angular/core";
+import {BehaviorSubject} from "rxjs";
+import {Cartesian3} from "cesium";
+
+@Injectable({providedIn: 'root'})
+export class ViewService {
```
Comment (josephbirkner) [inline erdblick_app/app/view.service.ts:6]: Please add a comment here, explaining why this service is needed. Something like

/**
 * The view service is required to buffer Cesium view settings, which may be set via the 
 * URL, before the actual Cesium view is available.
 */ 

I wonder though, why can't this be merged with ParameterService?

**Comment by Waguramu** [comment]
- [ ] Remove MapService in favour of ParametersService
- [ ] Remove ViewService in favour of ParametersService
- [ ] Sort .ts files into directories?
- [ ] Test whether everything works

**Review by josephbirkner — APPROVED** [review]
LGTM :)

**Pull request merged**